src/Microsoft.Data/DataFrameColumnCollection.cs

```diff
@@ -4,47 +4,54 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 
 namespace Microsoft.Data
 {
     /// <summary>
-    /// A DataFrameTable is just a container that holds a number of DataFrameColumns. It mainly acts as a convenient store to allow DataFrame to implement its algorithms
+    /// A DataFrameColumnCollection is just a container that holds a number of DataFrameColumns. It mainly acts as a convenient store to allow DataFrame to implement its algorithms
```

Comment:
> It mainly acts as a convenient store to allow DataFrame to implement its algorithms

Is this something users should know/care about? To users, this is a collection of DataFrameColumn instances.